### ISSUE TITLE
Connecting to Hana for SSAS from Visual Studio

### ISSUE THREAD

**Issue by romerohe** [MicrosoftDocs/powerbi-docs#2481]
I'm following the steps to connect to a hana database installing Hana Client, but it dind't work when I try to connect using Visual Studio in order to create a tabular model.

Despite of that, I do can connect to Hana from Power BI... Is there any additional steps to make it work in Visual Studio?

Thanks in advance

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ea414a5c-c3ca-281e-46ca-3e677d27924b
* Version Independent ID: 363a9c9c-13e2-58ce-cb79-9f897d83c307
* Content: [Use SAP HANA in Power BI - Power BI](https://docs.microsoft.com/en-us/power-bi/connect-data/desktop-sap-hana)
* Content Source: [powerbi-docs/connect-data/desktop-sap-hana.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/live/powerbi-docs/connect-data/desktop-sap-hana.md)
* Service: **powerbi**
* Sub-service: **powerbi-desktop**
* GitHub Login: @davidiseminger
* Microsoft Alias: **davidi**

**Comment by maggiesMSFT** [comment]
Hi, @romerohe -- interesting question! Assigning to the author to investigate. 
In the meantime, I recommend asking the Power BI community, https://community.powerbi.com/. 
Hope that helps!

**Comment by romerohe** [comment]
Hi Maggie, finally I could solve this issue... it costs me a whole morning
but it's worth it.

The origin of the issue is that I had installed the 64 bit version of Hana
Client, which matched perfectly with my Power BI 64 bits, but the ODBC
driver could work well with Visual Studio 32 bit.

The solution? I looked for Hana Client 32 bit and I have installed it as
well. It worked like a charm...

Maybe it is a good idea to mention this within the documentation because
maybe there are people with 32 bit version of Power BI with the same
problems... It is a thing to take care about.

Regards,
Hernán
